app/helpers/runs_helper.rb

```diff
@@ -41,4 +55,18 @@ def status_success_display(bool)
       "<span class=\"status-pending\">Pending</span>"
     end
   end
+
+  def carousel_playback_display(bool)
+    if bool.is_a? TrueClass
+      link_to class: "btn btn-browser-test-status" do 
+        "<span class=\"halflings halflings-repeat\"></span>".html_safe
+      end
+    else
+      link_to class: "btn btn-browser-test-status" do
+        "<span class=\"halflings halflings-trash\"></span>".html_safe
+    end
+    end
+  end
```

Comment:
I didn't like this method in the end and had already moved the functionality inside carousel_playback_button method. 

I thought I had already deleted this, I will update the PR.